src/ui-kit/form-controls/upload-v2/upload-v2.component.ts

```diff
@@ -287,12 +292,16 @@ export class SamUploadComponentV2 implements ControlValueAccessor {
     this.emit();
   }
 
-  initilizeFileCtrl(
-    {name, size, url, icon, disabled},
-    isSecure = false,
-    date = moment().format('MMM DD, YYYY h:mm a')) {
+  initilizeFileCtrl({name, size, url, icon, disabled, isSecure, postedDate}) {
```

Comment:
@kevin-song-reisys You need to make sure this change is coordinated with any other teams that are using this component. Please give them a heads up.

Reply:
Sure Colin, I'll send out a notice. Though I believe this is a non-breaking change as initializeFileCtrl is only used in one place, by the upload component, like this: `this.initilizeFileCtrl(uploadFile.file)`, so we were not passing anything for those optional parameters anyway. (https://github.com/GSA/sam-ui-elements/blob/master/src/ui-kit/form-controls/upload-v2/upload-v2.component.ts#L589)

Reply:
No issues for Octo